readme.md

```diff
@@ -38,6 +38,40 @@ Must be run in a separate terminal when the Docker environment is fully up and r
 docker exec -it app npm run test-docker --prefix backend    # Run tests in development env
 ```
 
+## Map structure
+<Project Root>
```

Comment:
Remove '<>' to ensure the tree is rendered properly  
Insert `  ` (double space) after each line except the last as well
```suggestion
Project Root```